openedx/core/djangoapps/programs/utils.py

```diff
@@ -20,3 +21,27 @@ def programs_api_client(api_url, jwt_access_token):
         api_url,
         jwt=jwt_access_token
     )
+
+
+def is_cache_enabled_for_programs():    # pylint: disable=invalid-name
+    """ Returns a Boolean indicating programs api used cached values or not."""
```

Comment:
"Returns a Boolean indicating whether responses from the Programs API will be cached."